gravityforms-regex-validation.php

```diff
@@ -82,7 +82,7 @@ function ToggleInputRegEx(isInit){
   // validate submitted data against provided regex
   public function validate($result, $value, $form, $field) {
     // if validation has passed so far, and regex validation is enabled, and a pattern was provided, and a value was provided
-    if ($result['is_valid'] && $field['regexValidation'] && !empty($field['regexPattern']) && !empty($value)) {
+    if ($result['is_valid'] && $field['regexValidation'] && !empty($field['regexPattern']) && ( strlen($value) != 0 )) {

```

Comment:
Good suggestion @leadoux! This is a little cleaner, and should work as well--look good to you?

```suggestion
    if ($result['is_valid'] && $field['regexValidation'] && !empty($field['regexPattern']) && strlen($value)) {
```